packages/core/src/__mocks__/connector.ts

```diff
@@ -14,7 +14,7 @@ export const mockMetadata: ConnectorMetadata = {
     'pt-PT': 'Conector',
     'zh-CN': '连接器',
     'tr-TR': 'Connector',
-    'ko-KR': 'Connector',
+    ko: 'Connector',
```

Comment:
should this be considered as a breaking change?

Reply:
No, we add the `ko`, not replace the `ko-KR`. and the old one still works.

Reply:
This change is only in the test cases.